### PR TITLE
Consolidates jellypbc.com and poster.jellypbc.com JEL-80

### LINKED ISSUE #226: Consolidate jellypbc.com and poster.jellypbc.com
we are consolidating jellypbc.com with poster.jellypbc.com

- jelly blog moves to a collection hosted at jellypbc.com/blog 
- move all blog posts as posts and add to a collection https://poster.jellypbc.com/@cindy/tags/jelly-blog
- fix reading experience for mobile devices
- add slack link and newsletter link to bottom of every blog post
- jellypbc.com homepage becomes our homepage (replacing request early access page)
- everything hosted at jellypbc.com (poster.jellypbc.com redirects to jellypbc.com)

### PR DESCRIPTION
**Pull request by cindywu**
fixes #226

we are consolidating jellypbc.com with poster.jellypbc.com

https://www.loom.com/share/39d6e86ba488475e9e6015fa3b0bfd18

 - [x] jelly blog moves to a collection hosted at jellypbc.com/blog
  - [x] move all blog posts as posts and add to a collection https://poster.jellypbc.com/@cindy/tags/jelly-blog
 - [x] jellypbc.com homepage include request early access, login link, slack, github, twitter
 - [x] add slack, twitter, github, blog links to bottom of every blog post https://poster.jellypbc.com/@cindy/tags/jelly-blog
 - [x] add jelly blog to homepage
 - [x] add newsletter signup to bottom of every blog post https://www.getrevue.co/profile/jelly
 - [x] /blog route goes to a collection called "jelly-blog" if it exists

future:
 - fix reading experience for mobile devices 
 - fix https://newsletter.jellypbc.com redirect to https://www.getrevue.co/profile/jelly
 - everything hosted at jellypbc.com (poster.jellypbc.com redirects to jellypbc.com)

note:

the rails app for the old jelly blog was moved from gitlab to github https://github.com/jellypbc/jellyblog

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -70,6 +70,8 @@
   get 'pricing', to: 'pages#pricing'
   get 'internship', to: 'pages#jobs'
 
+  get 'blog', to: redirect('@cindy/tags/jelly-blog')
```
Comment (dml-1) [inline config/routes.rb:73]: this should route to a specific action where we can render the collection, probably just pages#blog for now.

Comment (cindywu) [reply config/routes.rb:73]: @dml-1 could you start using conventional comments in reviews? https://conventionalcomments.org/

Comment (cindywu) [reply config/routes.rb:73]: i am not entirely sure what you mean, but i will try something and push up a change!

**Comment by cindywu** [comment]
> this should route to a specific action where we can render the collection, probably just pages#blog for now

@dml-1 could you review my new commit to see if the new changes addressed your request?

**Pull request merged**